#689: Use currencyCode instead of currencySymbol in price formatter

bonkey opened the pull request:
#689: Use currencyCode instead of currencySymbol in price formatter

bonkey commented:
I don't know/remember why I didn't use if from the beginning. Hopefully, it wasn't some edge case, which I don't remember now

hani-ibrahim commented:
That's interesting, I don't know if you noticed it or not 
but this:

![before](https://cloud.githubusercontent.com/assets/5626418/22059880/f5794c7e-dd6e-11e6-9eb3-68d4bee2ced2.png)

Is now changed to be this:

![after](https://cloud.githubusercontent.com/assets/5626418/22059879/f552a326-dd6e-11e6-9514-86e26823bc56.png)

Which is something I'm very happy with 👍 

I fixed the tests

hani-ibrahim commented:
👍

bonkey commented:
This was the only reason for this PR.

On Wed, 18 Jan 2017 at 11:13, Hani Ibrahim <notifications@github.com> wrote:

> That's interesting, I don't know if you noticed it or not
>
>
> but this:
>
>
> [image: before]
> <https://cloud.githubusercontent.com/assets/5626418/22059880/f5794c7e-dd6e-11e6-9eb3-68d4bee2ced2.png>
>
>
> Is now changed to be this:
>
>
> [image: after]
> <https://cloud.githubusercontent.com/assets/5626418/22059879/f552a326-dd6e-11e6-9514-86e26823bc56.png>
>
>
> Which is something I'm very happy with 👍
>
>
> I fixed the tests
>
>
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zalando-incubator/atlas-ios/pull/690#issuecomment-273436416>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACYvJmXEvDjUTELJQPsz5L3uT80MU-0ks5rTeXhgaJpZM4LmhzL>
> .
>
>
>
>
>
>
>
>
>
>
>
>
>
>
>
>
>
>
> --
Daniel Bauke, iOS Developer
Distributed Commerce, Zalando SE <http://tech.zalando.com>

bonkey commented:
👍
